specification/schemas/brokers/Broker.json

```
@@ -15,9 +15,12 @@
               "example": "MyParcel.com"
             },
             "domain": {
-              "type": "string",
+              "type": [
+                "string",
+                "null"
+              ],
               "maxLength": 255,
-              "example": "app.sendmyparcel.com"
+              "example": "app.myparcel.com"
```

Lets have a different example, like `example.com`, as in to motivate users that here they can enter their custom domain name. I think a description will be of use too.

```suggestion
              "example": "example.com",
              "description": "Set a custom domain for the backoffice. Defaults to app.myparcel.com."
```